iam-login-service/src/main/webapp/WEB-INF/views/iam/login-form.jsp

```diff
@@ -19,8 +19,22 @@
 <%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
 <%@ taglib prefix="t" tagdir="/WEB-INF/tags/iam"%>
 <%@ taglib prefix="spring" uri="http://www.springframework.org/tags"%>
-<form id="login-form" action="/login" method="post">
-  <div class="signin-preamble text-muted">Sign in with your ${iamOrganisationName} credentials</div>
+<c:if test="${loginPageConfiguration.defaultLoginPageLayout}">
+  <c:set value="container-spacer" var="containerSpacer"></c:set>
+</c:if>
+
+<form id="login-form" class="${containerSpacer}" action="/login" method="post">
+  <div class="signin-preamble text-muted">
+    <c:choose>
+      <c:when test="${loginPageConfiguration.defaultLoginPageLayout}">
+        Sign in with your ${iamOrganisationName} credentials
+      </c:when>
+      <c:otherwise>
+        Or sign in with your ${iamOrganisationName} credentials
```

Comment:
"Sign in" again?

Reply:
In our scenario, Assuming that user want external authn block to be displayed first, Now for our local credentials section, we have to change the text to `or sign in`. Does that make sense, ?